docs/_templates/deployments.html

```diff
@@ -0,0 +1,72 @@
+<h2> Cloud Computing Deployments </h2>
```

Comment:
One issue with hard coding this as HTML is that Sphinx doesn't parse the headings and put them in the table of contents. Can you think of a way around that?

Reply:
I noticed this. I didn't find a clean solution yet and didn't feel like it was a deal breaker. At first I wasn't splitting the list by cloud/hpc so it was even less of an issue.

Reply:
Ok nbd. Probably not worth the effort it would take to figure this out.